smtp/connection.ts

```diff
@@ -74,7 +74,7 @@ export type SMTPSocketOptions = Omit<
 >;
 
 export interface SMTPConnectionOptions {
-	timeout: number | null;
+	timeout: number | null | undefined;
```

Comment:
can we use `timeout?: number | null`

Reply:
see https://github.com/eleith/emailjs/pull/323#discussion_r981488358